CSS/styles.css

```diff
@@ -0,0 +1,882 @@
+  }
+
+  .campaign a {
+    color: var(--secondary);
+  }
+
+  /* Hero section */
+
+  .hero_container {
+    width: 100%;
+    margin: 0 auto;
+    padding-top: 2rem;
+    height: 90vh;
+  }
+
+  #hero {
+    margin: 0 auto;
+  }
+
+  .hero_heading {
+    width: 46.6rem;
+    margin: 0 auto;
+  }
+
+  .hero_introduction {
+    max-width: 47.3rem;
+  }
+
+  .hero_introduction p {
+    font-size: 25px;
+    font-weight: 400;
+    line-height: 36px;
+    letter-spacing: -0.01em;
+    text-align: left;
+  }
+
+  .venue-date {
+    width: 44.74rem;
+  }
+
+  .date {
+    font-family: 'Lato', sans-serif;
+    font-size: 42px;
+    font-weight: 900;
+    letter-spacing: -0.08em;
+    text-align: left;
+  }
+
+  .venue {
+    font-family: 'Lato', sans-serif;
+    font-size: 28px;
+    font-weight: 700;
+    letter-spacing: -0.02em;
+    text-align: left;
+  }
+
+  /* Program section */
+
+  .program h2 {
+    justify-content: center;
+    margin-top: 100px;
+  }
+
+  .program_head {
+    margin-top: 0;
+    text-align: center;
+  }
+
+  .program_container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .program_card {
+    width: 12.5rem;
+    height: 13.31rem;
+    padding: 0.375rem 1.125rem;
+    flex-direction: column;
+    margin-bottom: 90px;
+    margin-top: 30px;
+  }
+
+  .frame {
+    display: none;
+  }
+
+  /* speaker section */
+
+  .speakers {
+    height: 90vh;
+  }
+
+  .speaker_container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    max-width: 64.43rem;
+    height: 100%;
+    margin-bottom: 150rem;
+    padding-bottom: 10px;
+  }
+
+  .speaker {
+    max-height: 11.125rem;
+  }
+
+  .desk_ {
+    display: flex;
+  }
+
+  .more {
+    display: none;
+  }
+
+  /* Partner section */
+
+  #partner {
+    display: flex;
+  }
+
+  .partners img {
+    width: 8.5rem;
+  }
+
+  .footer_ {
+    display: block;
+    font-size: 16px;
+  }
+
+  /* About Us section */
+
+  #about_hero_container {
+    padding: 6.8125rem 1rem;
+    gap: 10px;
+    background: url(../Backgrounds/about_bg_main.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-color: rgba(247, 247, 248, 1);
+    margin: 10px auto;
+
+    /* height: 70vh; */
+  }
+
+  .about_hero {
+    text-align: center;
+
+    /* margin: 0 auto; */
+    padding: 10px;
+    font-size: 2.3rem;
+  }
+
+  .about_us_text {
+    max-width: 64%;
+
+    /* padding: 35px 65px 35px 65px; */
+    gap: 10px;
+    background: rgba(255, 255, 255, 1);
+    margin: 0 auto;
+  }
+
+  .about_us_text p {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 25px;
+    letter-spacing: 0;
+    text-align: center;
+    color: rgba(0, 0, 0, 1);
+  }
+
+  .frame_1 p {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 25px;
+    letter-spacing: 0.03em;
+  }
+
+  .creative_corner {
+    padding: 20px;
+  }
+
+  .about_logo {
+    margin-top: 20px;
+  }
```

Comment:
- [ ] To stay consistent with the layout of the original design kindly consider reducing the `width` of the image you have as your about logo as it will help make your design look more accurate. See below for ore details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="873" alt="Screenshot 2023-06-29 at 01 03 30" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/39d5572d-5aad-454d-81a8-a7e75ed019e5">
</td>
<td>
<img width="1440" alt="Screenshot 2023-06-29 at 01 04 41" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/e84669e0-7335-4714-8faa-ddc7ba9dbefa">
</td>
</tr>
</table>

Reply:
-  Hey, as pointed out by the previous reviewer, the image is still very big compared to the size of that in the about page.  please, kindly reduce it more to be close to the main design.